compiler/src/dotty/tools/dotc/typer/Checking.scala

```diff
@@ -992,10 +994,10 @@ trait Checking {
                 ("extractor", (n: Name) => s"prefix syntax $n(...)")
               else
                 ("method", (n: Name) => s"method syntax .$n(...)")
+            def rewriteMsg = Message.rewriteNotice("The latter", options = "-deprecation")
             report.deprecationWarning(
-              i"""Alphanumeric $kind $name is not declared `infix`; it should not be used as infix operator.
-                 |The operation can be rewritten automatically to `$name` under -deprecation -rewrite.
-                 |Or rewrite to ${alternative(name)} manually.""",
+              i"""Alphanumeric $kind $name is not declared ${hl("infix")}; it should not be used as infix operator.
```

Comment:
Unfortunately the syntax highlighting on `infix` has no effect, presumably because it is a soft keyword.

Reply:
could add some more utilities like:
```scala
def hlAsKeyword(str: String): String = s"${SyntaxHighlighting.KeywordColor}$str${SyntaxHighlighting.NoColor}"
```